### PR TITLE
Search By Users

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Search By Users

_Now searching for users with their username sis now possible

## How should this be tested?

1. Type the username of the user you are searching for in the NavSearch

Looks
image
image

image
image

## Linked issues
closes https://github.com/EveripediaNetwork/issues/issues/599

### CODE REVIEW

**Comment by kesar** [comment]
its not working by address

"we want to be able to place **an address** or an username and find the profile"

**Comment by OleanjiKingCode** [comment]
> its not working by address
> 
> "we want to be able to place **an address** or an username and find the profile"

Okay then ill work on the address own now

File: src/components/Layout/Navbar/NavSearch/index.tsx [hunk 1]
```diff
@@ -43,10 +43,12 @@ export type NavSearchProps = {
 const ItemPaths = {
   [SEARCH_TYPES.ARTICLE]: '/wiki/',
   [SEARCH_TYPES.CATEGORY]: '/categories/',
+  [SEARCH_TYPES.USERNAME]: '/account/',
```
Comment (kesar) [inline src/components/Layout/Navbar/NavSearch/index.tsx:46]: PR is full of word username when we want to refer to accounts, make sure you do a full review and replace it where needed. 👍🏻

Comment (OleanjiKingCode) [reply src/components/Layout/Navbar/NavSearch/index.tsx:46]: Alright

**Review by kesar — APPROVED** [review]

**Pull request merged**